robocorp-code/vscode-client/src/activities.ts

```diff
@@ -664,6 +664,7 @@ export async function createRobot() {
 export async function updateLaunchEnvironment(args) {
     let robot = args['targetRobot'];
     let environment = args['env'];
+    let task = args['task'] || 'latest-run';
```

Comment:
I think this may be confusing to the user if sometimes it's saved in one place and other times in another (so, I think that either we should always have a `task` or always use `latest-run` -- or use the previous approach for numbering runs).

Reply:
I added the latest-run only as a fallback if for some reason the task value is not provided. If you think it's better to make it a mandatory argument and throw an error in case task name is not provided, I think that would a good solution as well.

I asked about the numbered run vs task name as output in the demo and the design preferred the task name approach, I think it originally came from Janne anyway. I demoed that the next run with same task will overwrite the work item data and it was accepted.

Reply:
Ok, seems fair to make the task name mandatory then.

Reply:
As a note, there are cases where the task name is not available (when the run is made from the language server and not robocorp code) and using the task name is also dangerous (we need to escape characters because not all task names are valid filesystem entries depending on the OS).

So, I just left it as `latest-run`.

As we won't be cycling names, the work items library needs to be careful when it's rewriting the entries and dealing with locks on the open files for that on the filesystem level (but that'd be the same case even with the task name, just with a lower chance of happening).